ggml-quants.c

```diff
@@ -9452,7 +9452,7 @@ void ggml_vec_dot_iq3_s_q8_K (int n, float * GGML_RESTRICT s, size_t bs, const v
 
     const int nb = n / QK_K;
 
-#if defined(__ARM_NEON)
+#if defined(__ARM_NEON) && (__ARM_ARCH >= 8)
```

Comment:
Instead of this, we should add `ggml_vqtbl1q_u8` similar to how we have `ggml_vqtbl1q_s8` and replace all usages of `vqtbl1q_u8` with `ggml_vqtbl1q_u8`

Reply:
Amended